packages/kovan-faucets/src/ts/configs.ts

```diff
@@ -0,0 +1,10 @@
+export const configs = {
+    DISPENSER_ADDRESS: process.env.DISPENSER_ADDRESS.toLowerCase(),
+    DISPENSER_PRIVATE_KEY: process.env.DISPENSER_PRIVATE_KEY,
+    ENVIRONMENT: process.env.FAUCET_ENVIRONMENT,
+    ROLLBAR_ACCESS_KEY: process.env.FAUCET_ROLLBAR_ACCESS_KEY,
+    RPC_URL: process.env.FAUCET_ENVIRONMENT === 'development' ?
+        'http://127.0.0.1:8545' :
+        'https://kovan.infura.io/T5WSC8cautR4KXyYgsRs',
```

Comment:
Can we make the Infura API key a config? Just in case someone decides to clone this repo and run it?